### PR TITLE
release v0.1.25

### PR DESCRIPTION
**Pull request by andyh565**

### CODE REVIEW

File: packages/connectors/hull-hubspot/test/integration/scenarios/outgoing-account-update-validation-error-retry-failure.js [hunk 1]
```diff
@@ -421,10 +421,7 @@ it("should send out a new hull account to hubspot update validation error and re
           ["value","connector.service_api.response_time",expect.whatever()],
           ["increment","connector.service_api.error",1]
       ],
-      platformApiCalls: [
```
Comment (timliuhull) [inline packages/connectors/hull-hubspot/test/integration/scenarios/outgoing-account-update-validation-error-retry-failure.js:516]: Why are all of these platformApiCalls gone? just interested

File: packages/connectors/hull-hubspot-deals/manifest.json [hunk 2]
```diff
@@ -188,15 +207,16 @@
       }
     },
     {
-      "url": "/fields-outreach-prospect-out",
```
Comment (timliuhull) [inline packages/connectors/hull-hubspot-deals/manifest.json:191]: was this working before? seems a little weird, but looks right now

File: packages/connectors/hull-hubspot-deals/server/service.js [hunk 3]
```diff
@@ -122,11 +122,12 @@ const service = ({ clientID, clientSecret } : {
   //   }
   // },
   authentication: {
-    strategy: "hubspotOauth",
```
Comment (timliuhull) [inline packages/connectors/hull-hubspot-deals/server/service.js:125]: is the lower case right? just confirming, seems a little weird.  maybe just a difference in the latest code

File: packages/connectors/hull-hubspot/migrations/0.1.25-monorepo/reset-v0.1.25-monorepo.js [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+const _ = require("lodash");
+
+const ships = [
+  {
+    ship: "",
+    secret: "",
+    organization: ""
+  }
+];
+const parentDirectory = "/Users/hubspot";
```
Comment (timliuhull) [inline packages/connectors/hull-hubspot/migrations/0.1.25-monorepo/reset-v0.1.25-monorepo.js:15]: Cool! good thinking putting this in the migrations directory

Comment (timliuhull) [reply packages/connectors/hull-hubspot/migrations/0.1.25-monorepo/reset-v0.1.25-monorepo.js:15]: Love the rollback capability

File: packages/connectors/hull-hubspot/migrations/0.1.25-monorepo/v0.1.25-monorepo.js [hunk 5]
```diff
@@ -0,0 +1,392 @@
...
+const { defaultContactAttributes } = require("./default-contact-attributes");
+
+const ships = [
+  {
+    ship: "",
+    secret: "",
+    organization: ""
+  }
+];
+
```
Comment (timliuhull) [inline packages/connectors/hull-hubspot/migrations/0.1.25-monorepo/v0.1.25-monorepo.js:19]: Ouch, I feel your migration pain

File: packages/connectors/hull-hubspot/server/purplefusion/glue.js [hunk 6]
```diff
@@ -220,6 +220,7 @@ const glue = {
       set("stopFetchAt", ex(moment(), "valueOf")),
       loopL([
         set("contactsPage", hubspot("getRecentContactsPage")),
+        ifL(cond("isEmpty", "${contactsPage}"), loopEndL()),
```
Comment (timliuhull) [inline packages/connectors/hull-hubspot/server/purplefusion/glue.js:223]: Interesting case here that wasn't handled before

File: packages/connectors/hull-hubspot/test/integration/scenarios/outgoing-account-update-validation-error-retry-failure.js (hunk 1 above)
Comment (andyh565) [reply packages/connectors/hull-hubspot/test/integration/scenarios/outgoing-account-update-validation-error-retry-failure.js:516]: yeah I removed the need to get the user/account properties

File: packages/connectors/hull-hubspot-deals/manifest.json (hunk 2 above)
Comment (andyh565) [reply packages/connectors/hull-hubspot-deals/manifest.json:191]: hubspot deals wasn't set to auto deploy, so what version of master they were on, it worked, but not through this endpoint. I think handler name was automatically resolved

File: packages/connectors/hull-hubspot-deals/server/service.js (hunk 3 above)
Comment (andyh565) [reply packages/connectors/hull-hubspot-deals/server/service.js:125]: yeah, it's lowercase. Had to double check too

File: packages/connectors/hull-hubspot/migrations/0.1.25-monorepo/v0.1.25-monorepo.js (hunk 5 above)
Comment (andyh565) [reply packages/connectors/hull-hubspot/migrations/0.1.25-monorepo/v0.1.25-monorepo.js:19]: it was a pain...

File: packages/connectors/hull-hubspot/server/purplefusion/glue.js (hunk 6 above)
Comment (andyh565) [reply packages/connectors/hull-hubspot/server/purplefusion/glue.js:223]: this was causing an infinite loop. Performance looks soooo much better now

**Pull request merged**